app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LikesController < ApplicationController
+  def new
+    @post = Post.find(params[:post_id])
+    @user = User.find(params[:user_id])
+    @like = Like.new
+  end
+
+  def create
+    @user = User.find(params[:user_id])
```

Comment:
- [ ] It will be great to ensure that when you are creating a Like, the user, is the currently logged-in user. So just like you have done with the other models, get the user from the `current_user` method not from the params